src/components/chip-set/partial-styles/_file-picker.scss

```diff
@@ -79,3 +79,8 @@ $max-width-of-label-when-not-readonly: 40%;
         }
     }
 }
+
+.mdc-text-field.mdc-text-field--disabled.lime-text-field--readonly
+    .lime-looks-like-input-value {
+    padding-left: 1rem;
```

Comment:
this wouldn't work if there is no leading icon.
in file-picker there is a hardcoded leading icon. But chip-set can be with or without the leading icon.

Reply:
ahhh! right! thanks! will check the icon